essentials/src/api/dynamic.rs

```diff
@@ -71,6 +75,39 @@ pub(crate) fn decode_dynamic_scheduled_paras(raw_paras: &Value<u32>) -> Result<V
 	Ok(paras)
 }
 
+pub(crate) fn decode_dynamic_claim_queue(raw: &Value<u32>) -> Result<ClaimQueue, SubxtWrapperError> {
+	let decoded_btree_map = decode_unnamed_composite(raw)?;
+	let decoded_values = decode_unnamed_composite(decoded_btree_map.first().expect("Must contain values"))?;
+	let mut claim_queue: ClaimQueue = BTreeMap::new();
+	for value in decoded_values {
+		let decoded_tuple = decode_unnamed_composite(value)?;
+		let raw_core = decoded_tuple.first().expect("Must contain core");
```

Comment:
we should just return error and not panic when decoding values.

Reply:
That's a good remark. Thank you.